### PR TITLE
Update setup doc

### PR DESCRIPTION
**Pull request by R7L208**
Added additional information to setup instructions and included prerequisites within the setup instructions instead of a separate file. Deleted the existing prerequisites md file

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -8,37 +8,59 @@ description: |-
 
 # Databricks-Sync
 
-Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+Databricks-Sync is a synchronization utility for the export and import of Databricks-native objects from one workspace to another. The primary use case for Databricks-Sync is Disaster Recovery (incremental backup and restore); however, it can be used as a one-time migration utility.
```
Comment (itaiw) [inline docs/setup.md:11]: I think that "one-time" migration utility gives a wrong impression.
IMO we should position the tool for both migration and DR. you can think about it as a continuous migration.

Should we mention the CI/CD option as well?

File: docs/prerequisites.md [hunk 2]
```diff
@@ -1,31 +0,0 @@
-# Prerequisites
```
Comment (itaiw) [inline docs/prerequisites.md:1]: @R7L208 I think that we need these steps.
Is this on purpose?

File: docs/setup.md [hunk 3]
```diff
@@ -8,37 +8,59 @@ description: |-
...
+
+Please refer to the appropriate provider documentation for steps to verify SSH authentication.
 
 ## Dependencies
 
-This project requires the following environment dependencies:
+When using no public internet, Databricks-Sync requires the following environment dependencies:
 
 * [terraform 0.13.x](https://www.terraform.io/downloads.html)
 * [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
```
Comment (itaiw) [inline docs/setup.md:39]: Can we modify this to 0.3.x?

File: docs/setup.md [hunk 4]
```diff
@@ -8,37 +8,59 @@ description: |-
...
+* [GitLab](https://docs.gitlab.com/ee/ssh/)
+
+Please refer to the appropriate provider documentation for steps to verify SSH authentication.
 
 ## Dependencies
 
-This project requires the following environment dependencies:
+When using no public internet, Databricks-Sync requires the following environment dependencies:
 
 * [terraform 0.13.x](https://www.terraform.io/downloads.html)
```
Comment (itaiw) [inline docs/setup.md:38]: @stikkireddy should we add "or above"?

File: docs/setup.md [hunk 5]
```diff
@@ -8,37 +8,59 @@ description: |-
...
+* [Bitbucket Cloud](https://support.atlassian.com/bitbucket-cloud/docs/set-up-an-ssh-key/)
+* [Bitbucket Server](https://confluence.atlassian.com/bitbucketserver/enabling-ssh-access-to-git-repositories-in-bitbucket-server-776640358.html)
+* [GitLab](https://docs.gitlab.com/ee/ssh/)
+
+Please refer to the appropriate provider documentation for steps to verify SSH authentication.
 
 ## Dependencies
 
-This project requires the following environment dependencies:
+When using no public internet, Databricks-Sync requires the following environment dependencies:
```
Comment (itaiw) [inline docs/setup.md:36]: We need the TF installed whether you run online or offline

File: docs/setup.md (hunk 4 above)
Comment (stikkireddy) [reply docs/setup.md:38]: Ya a greater or equal to

File: docs/setup.md (hunk 1 above)
Comment (R7L208) [reply docs/setup.md:11]:
```suggestion
Databricks-Sync is a synchronization utility for the export and import of Databricks-native objects from one workspace to another. The primary use cases for Databricks-Sync are Disaster Recovery, migration, and CI/CD.
```

File: docs/setup.md (hunk 3 above)
Comment (R7L208) [reply docs/setup.md:39]:
```suggestion
* [terraform-provider-databricks 0.3.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
```

File: docs/setup.md (hunk 5 above)
Comment (R7L208) [reply docs/setup.md:36]:
```suggestion
Databricks-Sync requires the following environment dependencies:
```

File: docs/setup.md (hunk 1 above)
Comment (R7L208) [reply docs/setup.md:11]: @itaiw - does this sound better?

File: docs/setup.md (hunk 4 above)
Comment (R7L208) [reply docs/setup.md:38]: Not following this thread. Which dependency would this be for?

File: docs/setup.md [hunk 6]
```diff
@@ -8,37 +8,59 @@ description: |-
...
+Please refer to the appropriate provider documentation for steps to verify SSH authentication.
 
 ## Dependencies
 
-This project requires the following environment dependencies:
+Databricks-Sync requires the following environment dependencies:
 
 * [terraform 0.13.x](https://www.terraform.io/downloads.html)
-* [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
+* [terraform-provider-databricks 0.3.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
```
Comment (R7L208) [inline docs/setup.md:39]: Offline install is no longer being supported for terraform-provider-databricks.

File: docs/setup.md (hunk 4 above)
Comment (itaiw) [reply docs/setup.md:38]: * [terraform 0.13.x](https://www.terraform.io/downloads.html)

**Comment by R7L208** [comment]
Adding additional instructions for offline install then will re-open new PR

**Pull request closed without merging**